components/builder-protocol/protocols/originsrv.proto

```diff
@@ -218,6 +219,28 @@ message OriginPackageListRequest {
   optional bool distinct = 4;
 }
 
+message OriginPackagesForVersionRequest {
```

Comment:
This one actually represents a request by version (the request doesn't specify anything about channels), so in this case, I'd probably opt to leave this one as is, unless there's a better way to say it.